src/components/screens/Connect.tsx

```diff
@@ -67,10 +72,27 @@ export function Connect({ mode }: Props) {
   const [errorValue, setErrorValue] = useState<string | undefined>(undefined);
   const [scanner, setScanner] = useState(mode === "scan");
   const analytics = useAnalytics();
+  const core = useAtomValue(coreAtom);
   const web3wallet = useAtomValue(web3walletAtom);
   const pendingProposals = usePendingProposals(web3wallet);
   const sessions = useSessions(web3wallet);
   const showBackButton = pendingProposals.data.length || sessions.data.length;
+  const [_connectionStatus, setConnectionStatus] = useAtom(
+    relayerConnectionStatusAtom,
+  );
```

Comment:
You can use `useSetAtom`: https://jotai.org/docs/core/use-atom#usesetatom
```suggestion
  const setConnectionStatus = useSetAtom(
    relayerConnectionStatusAtom,
  );
```